db/template.go

```diff
@@ -18,6 +19,15 @@ func (d TinkDB) CreateTemplate(ctx context.Context, name string, data string, id
 		return errors.Wrap(err, "BEGIN transaction")
 	}
 
+	wf, err := pkg.ParseYAML([]byte(data))
```

Comment:
We should have a `tempate` package with a Parse method, not `pkg`. But this can be a separate issue/PR (#300)